README.md

```diff
@@ -97,6 +97,12 @@ When done, the container can be stopped with `docker stop target`.
   - `23.09`
   - `24.03-lts`
   - `24.09`
+- [openSUSE](https://ghcr.io/greenbone/vt-test-environments/opensuse) (`opensuse`)
+  - `leap:15.6`
```

Comment:
We can't use `:` in the tag. The full name would then be `...vt-test-environments/opensuse:leap:15.6`, which isn't allowed.
What do you think about dropping `leap` and `tumbleweed`, so that the tags are only called `opensuse:15.6` or `openSUSE:tumbleweed`?
There's every one one single `tumbleweed` release and other openSUSE products like Leap Micro and microOS only have very few or no security advisories available anyways. Besides that, only Leap and Tumbleweed are available on Docker Hub, so I the others are lower prio.